apps/challenges/views.py

```diff
@@ -147,8 +147,8 @@ def add_participant_team_to_challenge(request, challenge_pk, participant_team_pk
 @api_view(['POST'])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
+@permission_classes((IsChallengeCreator,))
 def disable_challenge(request, pk):
```

Comment:
@varunagrawal can you please add a test case for this view in which you test if the user is not the creator of the challenge then he is not allowed to disable the challenge. 

Let me know if you need some help with this.